tsp/src/async_store.rs

```diff
@@ -277,6 +283,89 @@ impl AsyncStore {
         Ok(())
     }
 
+    /// Send a nested relationship request to `receiver`, creating a new nested vid with `outer_sender` as a parent.
+    pub async fn send_nested_relationship_request(
```

Comment:
Should we move this logic to the 'sync' store?

Reply:
I would prefer to do this in a separate PR :-)

Reply:
Preferably in a seperate PR